var/spack/repos/builtin/packages/genomeworks/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Genomeworks(CMakePackage, CudaPackage):
+    """SDK for GPU accelerated genome assembly and analysis."""
+
+    homepage = "https://clara-parabricks.github.io/GenomeWorks/"
+    url      = "https://github.com/clara-parabricks/GenomeWorks/archive/v0.5.3.tar.gz"
+    git      = "https://github.com/clara-parabricks/GenomeWorks.git"
+
+    version('0.5.3', tag='v0.5.3', submodules=True)
+    version('0.5.2', tag='v0.5.2', submodules=True)
+    version('0.5.1', tag='v0.5.1', submodules=True)
+    version('0.5.0-rc1', tag='v-.5.0-rc1', submodules=True)
+    version('0.5.0', tag='v0.5.0', submodules=True)
+    version('0.4.4', tag='v0.4.4', submodules=True)
+    version('0.4.3', tag='v0.4.3', submodules=True)
+    version('0.4.0', tag='v0.4.0', submodules=True)
+    version('0.3.0', tag='v0.3.0', submodules=True)
+    version('0.2.0', tag='v0.2.0', submodules=True)
+
+    depends_on('cmake@3.10.2:', type=('build'))
+    depends_on('cuda@11:', type=('build', 'run'))
+    depends_on('python@3.6.7:', type=('build', 'run'))
+
+    phases = ['cmake', 'install']
```

Comment:
Do the default phases (cmake/build/install) not work?

Reply:
I don't know if the default works, but I'm following the instructions from https://github.com/clara-parabricks/GenomeWorks `make` is not mentioned, and I don't want this package to break because I assumed that `make` should be part of the installation process. I prefer to implement the exact documented steps. Is that correct?

Reply:
`phases` are not meant to be overridden in the majority of cases, since they are specific to each build system. Usually in CMake `make install` in the generated Makefiles implies a `make`, therefore the default will do. Can you delete this line too?

Reply:
Testing without. If it works, I'll update the PR.